transform/mattermost-analytics/models/staging/mm_calls_test_go/stg_mm_calls_test_go__tracks.sql

```diff
@@ -0,0 +1,22 @@
+WITH tracks AS (
+    SELECT
+        {{ dbt_utils.star(ref('base_mm_calls_test_go__tracks')) }}
+    FROM
+        {{ ref ('base_mm_calls_test_go__tracks') }}
+)
```

Comment:
No need for the CTE, just select the columns that are interesting in the main SELECT statement.

Reply:
Updated this, removed CTE